### DISCUSSION TITLE
Can't view ACSM file from google play

### DISCUSSION THREAD

**Discussion by kompre** [Leseratte10/acsm-calibre-plugin#80] [Q&A] [answered]
hey there,

I'm trying to view a file acsm that I exported from google play. I've dropped the file in calibre and it have read the metadata correctly and show a size of 1.5 MB, which tells me the plugin must have worked somehow, berceuse before installing it (and noDRM) the entry read only the name of the file and size less than 0.1 MB.

But when I try to view the book, it says is locked by drm. 

- the imported book:

![image](https://github.com/Leseratte10/acsm-calibre-plugin/assets/11796345/5773ad02-4f94-4d96-8333-2fd711edbad9)

- the error message:

![image](https://github.com/Leseratte10/acsm-calibre-plugin/assets/11796345/b56b8992-7912-4074-bdc9-1688dcdf1c87)

It means the plugin is not able to take off the drm, or I'm missing something?

**Comment by Leseratte10** [comment, marked as answer]
Correct, this looks like my plugin did its job successfully but the DRM removal failed. 

Try exporting the DER file from my plugin's settings and manually import it into DeDRM. Also, make sure you're on the most recent version of the DeDRM plugin. 

If that still doesn't help you'll need to open a bug report for the DeDRM plugin as this doesn't seem to be an issue with my plugin.

**Reply by kompre** [reply]
This actually worked. 

The first time I opened the DeDRM plugin, I've seen a key added and assumed it was the one generated by your plugin. I deleted that one entry and manually imported the DER file and now it works (if matters, the original entry and the imported one had different name).

I think I have fumbled a bit though: I've tried to de-drm on two different machine, with different Anonymous generated code. I can see that that one acsm I successfully imported in first machine, cannot be imported in the second machine, and viceversa. I suppose this means the acsm file is now linked to either anonymous code and cannot be imported by any other different installation with different DER code.

I really need to be more careful how do import them and care really care of the code otherwise I could really lose access to them.

OK now I understand a little better how DRM works.

**Comment by kompre** [comment]
Thank you, I'm very new to this stuff.

Does it matter that I've used an anonymous Adobe digital ID? Maybe it's just for testing purpose?

**Reply by Leseratte10** [reply]
No, an anonymous ID should work just fine.
